doc/changes/9078.md

```diff
@@ -0,0 +1,2 @@
+- Fix incorrect parsing of `dune-package` files in the presence of installed
```

Comment:
Since the feature is not part of a released version I think that this PR can just amend `doc/changes/package-dirs.md`

Reply:
I just got rid of this. If it wasn't released, no need to mention it.